### DISCUSSION TITLE
Can't disable `S` when in visual mode

### DISCUSSION THREAD

**Discussion by LintaoAmons** [folke/flash.nvim#153] [Q&A]
I only want `S` enabled in normal mode, but seems can't right now.
I switch the `Flash` version back to `v1.1.0`, it works. So I think this might be a bug with the new version?

(Because I want to use `S` to perform `surrounding` action in visual mode)
Here's my config
```lua
return {
  "folke/flash.nvim",
  event = "VeryLazy",
  -- tag = "v1.1.0",
  ---@type Flash.Config
  opts = {},
  keys = {
    {
      "s",
      mode = { "n", "x", "o" },
      function()
        -- default options: exact mode, multi window, all directions, with a backdrop
        require("flash").jump()
      end,
    },
    {
      "S",
      mode = { "n" },
      function()
        require("flash").treesitter()
      end,
    },
    {
      "r",
      mode = "o",
      function()
        require("flash").remote()
      end,
      desc = "Remote Flash",
    },
  },
}
```

**Comment by folke** [comment]
What do you mean? What is not working? Your keymap is only for normal mode, so that's what you wanted?

**Reply by LintaoAmons** [reply]
Yes, I want `S` to only work at normal mode, but the most recent version, with my current configuration, it will still be triggered in virtual mode.

But with v1.1.0, it will only be triggered in normal mode (expected behaviour as per the configuration)

**Reply by folke** [reply]
But you only map it to normal mode, so that's just not possible. Or are you using LazyVim maybe? If so, check the lazyvim docs on how to properly disable and override plugin keymaps
